plasmapy/physics/parameters_cython.c

```diff
@@ -1,5 +1,20 @@
 /* Generated by Cython 0.27.3 */
 
+/* BEGIN: Cython Metadata
```

Comment:
That's interesting. Why did this metadata suddenly appear?

Reply:
It actually appeared a while ago for me, I kept deselecting this file to be committed in PyCharm.  Guess I forgot sometime along the way in this PR. I'm not quite sure, actually.  It seems to claim that Cython's version didn't change, so... no idea.

Reply:
Yea, it's fine if it stays there. I was just curious.